Issue with accented chars in file or directory on docker

**Describe The Bug**
A clear and concise description of what the bug is. Please include as much detail as possible.

**To Reproduce The Bug**
Install as a docker

Choose Import

Select the folder to scan

All folders with accented characters are ignored and the file dont show. All files with an accented characters but not in a folder with accented characters are shown but throws an error on import

I'm french so accented chars are often presents. I tested with the windows version and there is no issue all files are retreived. The issue is only on the docker version (i guess it's related with maybe the LANG env var which i think should be a param)

Here is the error on the case of files

Caused by: java.io.FileNotFoundException: /mediatheque/Ebooks/A traiter/Bds/My Hero Academia (2016)/My Hero Academia T22 - C201 ?? 212 [168p] (Kouhei HORIKOSHI) [Scantrads].cbr (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method) ~[na:na]
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219) ~[na:na]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157) ~[na:na]
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112) ~[na:na]
	at org.comixedproject.adaptors.file.FileTypeAdaptor.getArchiveAdaptorFor(FileTypeAdaptor.java:72) ~[comixed-adaptors-1.2.0-1.jar!/:1.2.0-1]


For folder it's the same but as the folder is considered non existing due to the chars it's content is totaly ignored

**Expected Behavior**
All files should be imported wheter they contain an accented characters or wheter they are in a folder with an accented characters. 



 - Docker image Official
 - Browser name not related to browser


**ComiXed Build Detail:**
Le code a été build depuis la branche nommée bf522ad183f411116649c0a7d46ead0e0393a478
Le code a été build le October 1, 2022 at 1:17:56 PM GMT+2.
Le code a été build sur la machine nommée fv-az173-109
Le version du code est : 1.2.0-1.
Le hash du dernier commit de la branche source est bf522ad183f411116649c0a7d46ead0e0393a478.
Le commit a été créé le October 1, 2022 at 12:50:49 AM GMT+2.
Message du commit
Changed the release version to 1.2.0-1 [#1418]
Le commit a été créé par Darryl L. Pierce <mcpierce@gmail.com>
La branche ne contient pas de modification non comité.
Cette branche suit la branche distante https://github.com/comixed/comixed
L'url de la base de données utilisée est jdbc:h2:file:~/.comixed/comixed;create=true

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,8 +1,19 @@
 FROM debian:buster-slim
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-       openjdk-11-jre unzip wget bash
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends \
+       openjdk-11-jre \
+       unzip \
+       wget \
+       bash \
+       locales \
+       locales-all
+RUN rm -rf /var/lib/apt/lists/*
+RUN locale-gen "en_US.UTF-8"
+
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
 
 MAINTAINER The ComiXed Project "comixed-dev@freelists.org"
 
@@ -18,5 +29,6 @@ ENV PATH="$JAVA_HOME/bin:${PATH}"
 
 EXPOSE 7171
 VOLUME /comic_data
+VOLUME /library
 
 CMD ["bash", "/app/comixed-release-1.2-SNAPSHOT/bin/run.sh", "-L", "/library/comixed.log"]

